### PR TITLE
Updates testing config for rover

### PR DESCRIPTION
**Pull request by akeem**
# Description

Adds configuration allowing test to actually run, and library missing to ensure that test do indeed pass.

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: rover/src/storage.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import "reflect-metadata";
```
Comment (cellog) [inline rover/src/storage.ts]: rebase needed?

Comment (akeem) [reply rover/src/storage.ts]: this is actually being used by typeorm https://stackoverflow.com/questions/49618719/why-typeorm-needs-reflect-metadata

Comment (cellog) [reply rover/src/storage.ts]: no I mean this is not a configuration file, is it? (storage.ts)

Comment (cellog) [reply rover/src/storage.ts]: (I thought I saw it in a previous PR)

Comment (akeem) [reply rover/src/storage.ts]: this file was missing, its being added here to ensure that the test don't break when the configure is applied

**Pull request merged**